sdk/tables/azure-data-tables/samples/async_samples/sample_insert_delete_entities_async.py

```diff
@@ -75,7 +77,7 @@ async def delete_entity(self):
         from azure.core.exceptions import ResourceNotFoundError, ResourceExistsError
         from azure.core import MatchConditions
 
-        table_client = TableClient(account_url=self.account_url, credential=self.access_key, table_name=self.table_name)
+        table_client = TableClient(account_url=self.account_url, credential=AzureNamedKeyCredential(self.account_name, self.access_key), table_name=self.table_name)
```

Comment:
For the samples, can we move the instantiation of the credential out onto a separate line?
Helps with readability.